Add SunFounder PiPower 3 dtoverlay

cavonlee opened the pull request:
Add SunFounder PiPower 3 dtoverlay
edit SunFounder Pironman 5 IR default pin to 13

pelwell commented:
You can ignore some of the the checkpatch errors, but using TABs for consistent indentation of the overlay source would be appreciated. The dtoverlaycheck errors are a red line: https://github.com/raspberrypi/linux/actions/runs/9545679078/job/26307040713?pr=6227

cavonlee committed:
feat: Add support for SunFounder PiPower 3 overlay
fix: Fix wrong Pironman 5 ir default pin number
fix: Change space indentation to tab

Signed-off-by: Cavon Lee <cavonxx@gmail.com>

cavonlee commented:
@pelwell Thank you, I'll try to fix it all
